File: kotlin/selection_sort.kt
fun SelectionSort(arr: Array<Int>): Array<Int> {
  var i = 0
  while (i < arr.size) {
    var min = i
    var j = i + 1
    while (j < arr.size) {
      if (arr[j] < arr[min]) {
        min = j
      }
      j++
    }
    if (min != i) {
      var temp = arr[i]
      arr[i] = arr[min]
      arr[min] = temp
    }
    i++
  }
  return arr
}

File: php/selection_sort.php
<?php
function selection_sort($arr)
{
  for ($i = 0; $i < count($arr); $i++) {
    $min = $i;
    for ($j = $i + 1; $j < count($arr); $j++) {
      if ($arr[$j] < $arr[$min]) {
        $min = $j;
      }
    }
    if ($i !== $min) {
      $temp = $arr[$i];
      $arr[$i] = $arr[$min];
      $arr[$min] = $temp;
    }
  }
  return $arr;
}